api/src/resources/entity_def.rs

```diff
@@ -23,3 +23,17 @@ pub struct NewEntityDef {
     pub title: String,
     pub trainable: bool,
 }
+
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+pub struct NewGeneralFieldDef {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub field_type_id: Option<String>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub title: Option<String>,
```

Comment:
`title`s for General Fields are deprecated, we're only operating on `api_names` going forward. If you look at what the webapp does, it's actually sending a random alphanumeric as the title and there is no way to set the title from the ui

![image](https://github.com/reinfer/cli/assets/11478679/c125fb47-ca3b-4ecd-b932-6b416e01b7b4)